lib/chef/win32/security.rb

```diff
@@ -632,7 +647,19 @@ def self.has_admin_privileges?
 
           true
         else
-          process_token = open_current_process_token(TOKEN_READ)
+          # a regular user doesn't have privileges to call Chef::ReservedNames::Win32::Security.OpenProcessToken
+          # hence we return false if the open_current_process_token fails with `Access is denied.` error message.
+          begin
+            process_token = open_current_process_token(TOKEN_READ)
+          rescue Exception => run_error
+            return false if run_error.message =~ /Access is denied/
```

Comment:
Will this cause a problem for non english locales? I'm guessing chef generally doesn't work in those cases anyway?

Reply:
When `false` is returned from this method (`has_admin_privileges?`), then we raise a customized error message. If this fails for a non English locale, then the next line `Chef::ReservedNames::Win32::Error.raise!` will raise the error.
So this won't particularly break anything.